registry/infrastructure/repositories/organization_repository.py

```diff
@@ -145,3 +161,26 @@ def _update_org_status(self, org_workflow_model, status, username):
         org_workflow_model.OrganizationReviewWorkflow.status = status
         org_workflow_model.OrganizationReviewWorkflow.updated_on = current_utc_time
         org_workflow_model.OrganizationReviewWorkflow.updated_by = username
+
+    def add_group(self, groups, org_uuid):
+        organizations = self.get_org_for_uuid(org_uuid)
+        if len(organizations) > 0:
+            latest_org_record = None
+            for org in organizations:
+                if org.OrganizationReviewWorkflow.status == OrganizationStatus.DRAFT.value:
+                    latest_org_record = org
+            if latest_org_record is None:
+                pass
+            else:
+                org_row_id = latest_org_record.Organization.row_id
+                self.session.query(Group).filter(Group.org_row_id == org_row_id).delete()
+                self.session.commit()
+                groups = [Group(
+                    org_row_id=org_row_id,
+                    org_uuid=org_uuid, id=group.group_id, name=group.name,
+                    payment_config=group.payment_config, payment_address=group.payment_address, status=""
+                ) for group in groups]
+                self.add_all_items(groups)
+                self.session.commit()
+        else:
```

Comment:
we need to throw an error if someone is adding a group to a non-existing organization